battleship-client/src/test/java/com/academy/solid/nie/client/ui/BoardTest.java

```diff
@@ -24,7 +23,7 @@ public void shouldPlaceShip_whenThereAreNoShips() {
         EventHandler<MouseEvent> handler = Mockito.mock(EventHandler.class);
         board.initialize(handler);
 
-        new Cell(point).addShip(ship);
+        new Cell(point).setShip(ship);
```

Comment:
Why do you need this call? It's not used, you're not capturing the reference. Aside from using CPU and RAM this does nothing?

Reply:
Bump.